vue.config.js

```diff
@@ -1,5 +1,36 @@
+const webpack = require('webpack')
+
 module.exports = {
-  transpileDependencies: [
-    'vuetify'
-  ]
+  // Do not put unique hash in generated bundle name
+  filenameHashing: false,
+  configureWebpack: {
+    // Name the bundle 'volunteer-tasks' (dist/js/volunteer-tasks.js)
+    entry: {
+      'volunteer-tasks': './src/main.ts'
+    },
+    plugins: [
+      // Do not split the bundle up into chunks (generate 1 file)
+      new webpack.optimize.LimitChunkCountPlugin({
+        maxChunks: 1
+      })
+    ]
+  },
+  chainWebpack: config => {
+    // Delete the always-generated bundle
+    // https://github.com/vuejs/vue-cli/issues/2969
+    config.optimization.delete('splitChunks')
+    config.entryPoints.delete('app')
+  },
+  devServer: {
+    proxy: {
+      // proxy all /api requests to localhost:3000 to avoid CORS issues
+      '/api': {
+        target: `http://localhost:${process.env.PORT || 3000}`,
+        changeOrigin: true,
+        pathRewrite: {
+          '^/api': ''
+        }
+      }
+    }
+  }
```

Comment:
We may want to enable hot-reloading here at some point - our app is simple enough that it should work well without too many weird issues

Reply:
Hot-reloading is actually enabled here! 😎 This webpack configuration overrides Vue's builtin bundle config which includes hot-reloading.